qtt/algorithms/coulomb.py

```diff
@@ -23,31 +19,35 @@
 
 import scipy.optimize as opt
 
-# http://stackoverflow.com/questions/10582795/finding-the-full-width-half-maximum-of-a-peak
-
 
 def gauss(x, p):
     """ Gaussian function with parameters
-        p[0]==mean, p[1]==stdev, p[2]=scale
+
+    Args:
+        x (array or float): input variable
+        p (array): parameters [mean, std. dev., amplitude]
+    Returns:
+        array or float: calculated Gaussian
     """
     return p[2] * 1.0 / (p[1] * np.sqrt(2 * np.pi)) * np.exp(-(x - p[0])**2 / (2 * p[1]**2))
 
 
-def analyseCoulombPeaks(all_data, fig=None, verbose=1):
+def analyseCoulombPeaks(all_data, fig=None, verbose=1, parameters={}):
```

Comment:
This might be dangerous, since a dictionary is mutable. More info see [here](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html).